pkgs/applications/networking/cluster/crc/default.nix

```diff
@@ -0,0 +1,63 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, git
+, stdenv
+, testers
+, crc
+, runtimeShell
+, coreutils
+}:
+
+buildGoModule rec {
+  pname = "crc";
+  version = "2.4.1";
+  gitCommit = "6b954d4";
```

Comment:
```suggestion
```
This must be manually updated which adds a bit of overhead. Since it is purely informational we can just hardcode it to ``v${version}``.